test/unit/v0/test_account_info.py

```diff
@@ -184,8 +201,12 @@ def _test_account_info(self, check_persistence):
             self.assertEqual('account_auth', info2.get_account_auth_token())
             self.assertEqual('api_url', info2.get_api_url())
             self.assertEqual('app_key', info2.get_application_key())
+            self.assertEqual('key_id', info2.get_application_key_id())
             self.assertEqual('realm', info2.get_realm())
             self.assertEqual(100, info2.get_minimum_part_size())
+            self.assertTrue(info2.is_same_key('key_id', 'realm'))
+            self.assertFalse(info2.is_same_key('key_id', 'another_realm'))
+            self.assertFalse(info2.is_same_key('another_key_id', 'realm'))
```

Comment:
```suggestion
            self.assertFalse(info2.is_same_key('another_key_id', 'realm'))
            self.assertFalse(info2.is_same_key('another_key_id', 'another_realm'))
```
The bug this additional test case protects from would be pretty weird, but I'd write it